components/service/glean/src/test/java/mozilla/components/service/glean/histogram/FunctionalHistogramTest.kt

```diff
@@ -0,0 +1,68 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.service.glean.histogram
+
+import androidx.test.core.app.ApplicationProvider
+import mozilla.components.service.glean.testing.GleanTestRule
+import org.junit.Assert.assertEquals
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.RobolectricTestRunner
+import java.lang.Math.pow
+
+@RunWith(RobolectricTestRunner::class)
```

Comment:
Can this be done with AndroidJUnit test runner?  I know that the rest of Glean still has a lot of Robolectric but the rest of android-components has already switched over.